CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- Fix failing `type object ... has no attribute '_is_protocol'` on Python 3.7
```

Comment:
Feel free to give yourself credit in the CHANGELOG entry, as well!

```suggestion
- Fix regression in version 4.6.1 where comparing a generic class against a 
  runtime-checkable protocol using `isinstance()` would cause `AttributeError`
  to be raised if using Python 3.7
```